src/api/app/views/webui2/webui/staging/projects/_history_elements.html.haml

```diff
@@ -2,8 +2,14 @@
   - elements.each do |element|
     %li
       = element.event_type.humanize
-      = link_to("##{element.bs_request.number}", request_path(element.bs_request.number))
-      for package
-      %b= element.package_name
-      submitted by
-      %b= element.user_name
+      - case element.event_type
+      - when 'unstaged_request', 'staged_request'
+        = link_to("##{element.bs_request.number}", request_path(element.bs_request.number))
+        for package
+        %b= element.package_name
+        submitted by
+        %b= element.user_name
+      - when 'staging_project_created'
+        %b #{time_ago_in_words(Time.at(element.datetime.to_i))} ago
+        by
+        %b= element.user_name
```

Comment:
That's more of a nitpick... I would prefer to move this to a helper. So we don't end up with a big switch condition in the view.

Reply:
I was also considering if I should do it or not when I wrote the code. I will move it to a helper.

Reply:
Done, let me know what you think.